Makefile

```
@@ -62,6 +63,12 @@ run-django:
 	rm -rf .parcel-cache
 	honcho start -f Procfile.dev
 
+run-all:
+	docker compose --profile airflow up -d
+	rm -rf .parcel-cache
+	.venv/bin/python manage.py runserver 0.0.0.0:8000
```

```suggestion
	$(DJANGO_ADMIN) runserver 0.0.0.0:8000
```

J'ai défini un alias pour pouvoir faire évoluer ça plus facilement